modules/cert-manager-override-environment-variables.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// * security/cert_manager_operator/cert-manager-customizing-api-fields.adoc
+
+:_content-type: PROCEDURE
+[id="cert-manager-override-environment-variables_{context}"]
+= Customizing cert-manager by overriding environment variables from the cert-manager Operator API
+
+You can override the default environment variables for the {cert-manager-operator} by adding a `spec.controllerConfig` section in the `CertManager` resource.
+
+.Prerequisites
+
+* You have access to the {product-title} cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. Edit the `CertManager` resource by running the following command:
++
+[source,terminal]
+----
+$ oc edit certmanager cluster
+----
+
+. Add a `spec.controllerConfig` section with the following override arguments:
++
+[source,yaml]
+----
+apiVersion: operator.openshift.io/v1alpha1
+kind: CertManager
+metadata:
+  name: cluster
+  ...
+spec:
+  ...
+  controllerConfig: <1>
+    overrideEnv:
+      - 'HTTP_PROXY'
```

Comment:
I missed it in earlier reviews, the overrideEnv section should be of the format:
```
overrideEnv:
  - name: HTTP_PROXY
     value: http://<PROXY_URL>
  - name: HTTPS_PROXY
     value: https://<PROXY_URL>
   - name: NO_PROXY
     value: <IGNORE_PROXY_DOMAINS>
```
Where <proxy_url> is the url to the proxy server and <ignore_proxy_domains> are comma separated list of domains which proxy should ignore eg. “.example.com,*.local.net”.

xref: https://access.redhat.com/documentation/en-us/openshift_container_platform/3.4/html/installation_and_configuration/install-config-http-proxies

Reply:
Yes, this needs update. https://github.com/openshift/cert-manager-operator/blob/master/deploy/examples/cluster-cert-manager-overrides.yaml shows format example as well